README.md

```diff
@@ -64,6 +64,7 @@ These are real examples using the code *isogram* generates.
 | gulp.js                       | [gulpjs.com][gulp]        | `g` `u` `l` `p` `j` `s`     |
 | hapi                          | [hapijs.com][hapi]        | `h` `a` `p` `i` `j` `s`     |
 | Jekyll                        | [jekyllrb.com][jekyll]    | `j` `e` `k` `y` `l` `L`     |
+| Jekyll                        | [kanyewest.com][builtbylane]    | `k` `a` `n` `y` `e`     |
```

Comment:
Currently your PR uses wrong markdown syntax so it cannot be shown as you expected. https://github.com/shinnn/isogram/tree/31042992f81a916c0b6cb3e1f5e4fee5de221de9#sites-using-isogram Please update it to be consistent with other links.